src/main.c

```diff
@@ -201,195 +208,224 @@ int main()
       RELAY = 1;
 
       // run every ~1 secs
-      if (count % 4 == 0) {
+      if ((count & 3) == 0) {
```

Comment:
I need to upgrade my sdcc version, then, because the 3.3.0 was generating something with a 'div ab' instruction.